src/genet/core.py

```diff
@@ -784,6 +784,56 @@ def empty_modes(mode_attrib):
         )
         self.remove_links(no_mode_links)
 
+    def split_links_on_mode(self, mode: str, link_id_prefix: Optional[str] = None) -> Set[str]:
+        """Method to split links depending on mode.
+        Existing links with mode `mode` will have that mode removed.
+        New links will be added with only the mode `mode` and inheriting data from the link they originated from.
+        The IDs of new link IDs will by default identify the mode, but can be changed with `link_id_prefix`.
+
+        Examples:
+            ```python
+            [1] network.link("LINK_ID")
+            [out] {"id": "LINK_ID", "modes": {"car", "bike"}, "freespeed": 5, ...}
+            ```
+
```

Comment:
Am I misreading/misunderstanding this? It feels like the actual function call is missing from this example. I expected a before state, a call to the function, and an after state.